docs/source/examples/Notebook/rstversions/Typesetting Equations.rst

```diff
@@ -50,7 +50,7 @@ Source
     \end{equation*}
 
 Display
--------
+~~~~~~~
 
```

Comment:
FYI. The raw:: latex in the Display sections are not displaying locally. Not sure if it is a dependency issue locally on my system or something else.